src/Network/Network/Generated/LoadBalancer/LoadBalancerRule/NewAzureRmLoadBalancerRuleConfigCommand.cs

```diff
@@ -83,6 +84,11 @@ public partial class NewAzureRmLoadBalancerRuleConfigCommand : NetworkBaseCmdlet
             ValueFromPipelineByPropertyName = true)]
         public int IdleTimeoutInMinutes { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The timeout for the TCP idle codnnection. The value can be set between 4 and 30 minutes. The default value is 4 minutes. This element is only used when the protocol is set to TCP.")]
```

Comment:
The helpmsg should describe the meaning of EnableHighAvailability. Besides, you did not mention this parameter in your cmdlet design. Is the parameter necessary for new-AzureRmLoadBalancerRuleConfigCommand? If yes, could you add it to the design doc?

Reply:
yes we had to discuss this in our group and decided to remove EnableHighAvailability so it will not be needed